### ISSUE TITLE
Consulting for the loss function in LiDAR and Distribution Reward Modeling

### ISSUE THREAD

**Issue by DavidYan2001** [jinlong17/LightDiff#2]
Hi authors,

Thanks for your nice work!

When I read the paper, I find that the loss in LiDAR and Distribution Reward Modeling section is confusing. Can you explain more about the Reinforcement Learning (RL) training strategy utilized for the loss in the paper?

Hope to hear from you soon!
